kfdef/codeflare-stack-kfdef.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kfdef.apps.kubeflow.org/v1
+kind: KfDef
+metadata:
+  name: codeflare-stack
+  namespace: opendatahub
+spec:
+  applications:
+  # CodeFlare
```

Comment:
When deploying this KfDef, I get following error 
```
' (kubeflow.error): Code 500 with message: kfApp Apply failed for kustomize:  (kubeflow.error): Code 500 with message: error evaluating kustomization manifest for codeflare-stack: error mapping rest config no matches for kind "InstaScale" in version "codeflare.codeflare.dev/v1alpha1"'
```

Is there any pre-requisite for creating the KfDef?